images/dasklab/build.sh

```diff
@@ -10,12 +11,12 @@ IMAGE_NAME=$1
 
 time docker build -f dockerfile.build --target testing .
 time docker build -f dockerfile.build \
-    -t $DOCKER_REGISTRY/$IMAGE_NAME:$BUILD_TAG \
-    -t $DOCKER_REGISTRY/$IMAGE_NAME:latest \
-    -t $DOCKER_REGISTRY/$IMAGE_NAME:$COMMIT_HEAD \
-    --target release .
+	-t "$DOCKER_REGISTRY"/"$IMAGE_NAME":"$BUILD_TAG" \
```

Comment:
You can also do `"$DOCKER_REGISTRY/$IMAGE_NAME:$BUILD_TAG"`. I like it since it's easier to not miss a `"` if you ever edit it

Reply:
We're going to be migrating all these images to Actions -> GHCR, so my plan is to ignore these files until they go away :+1: :rofl: